src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
Everything works yeah. I added screenshots above to show.

I mentioned you can't use stage 0 features in the transform itself because the halting code (`halts(transformCode);`) we added it gives an unexpected token error (because it uses acorn to parse the code).